environment/aws/common/io.py

```diff
@@ -108,12 +108,23 @@ def unzip_directory(input: Path, output: Path) -> None:
 
     with zipfile.ZipFile(input, "r") as zipf:
         for member in tqdm(zipf.infolist(), desc="Unzipping"):
-            zipf.extract(member, output)
             extracted_path = output / member.filename
 
-            # Preserve file permissions
-            perm = member.external_attr >> 16
-            if perm:
-                extracted_path.chmod(perm)
+            # Check if the member is a symlink
```

Comment:
Where did this zipfile come from?
As I recall, the JAK e2e test server build doesn't depend on any symlinks...